### PR TITLE
Copy Edit troubleshooting.rst

### PR DESCRIPTION
**Pull request by peaceheis**
This is for Issue #1401.

## Links to added or changed documentation pages

https://manimce--2609.org.readthedocs.build/en/2609/,

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: docs/source/installation/troubleshooting.rst [hunk 1]
```diff
@@ -87,19 +89,19 @@ instructions given in
 of the corresponding `GitHub repository <https://github.com/ManimCommunity/ManimPango>`_.
 
 
-(Windows) OSError: dlopen() failed to load a library: pango?
+(Windows) OSError: dlopen() failed to load a library: pango
 ------------------------------------------------------------
```
Comment (behackl) [inline docs/source/installation/troubleshooting.rst:93]: The underline below a section header should always have the same length as the title itself.

```suggestion
-----------------------------------------------------------
```

File: docs/source/installation/troubleshooting.rst [hunk 2]
```diff
@@ -87,19 +89,19 @@ instructions given in
...
 
 This should be fixed in Manim's latest version, update
 using ``pip install --upgrade manim``.
 
 
 
-Some letters are missing from Text/Tex output?
+Some letters are missing from Text/Tex output
 ------------------------------------------------------------
```
Comment (behackl) [inline docs/source/installation/troubleshooting.rst:101]:
```suggestion
Some letters are missing from Text/Tex output
---------------------------------------------
```

File: docs/source/installation/troubleshooting.rst [hunk 3]
```diff
@@ -108,7 +110,7 @@ uses. Which can be done by running:
 
 .. _dvisvgm-troubleshoot:
 
-Installation does not support converting PDF to SVG?
+Installation does not support converting PDF to SVG
 ----------------------------------------------------
```
Comment (behackl) [inline docs/source/installation/troubleshooting.rst:114]:
```suggestion
---------------------------------------------------
```

File: docs/source/installation/troubleshooting.rst [hunk 4]
```diff
@@ -173,7 +175,7 @@ As a last check, you can run
 
   dvisvgm -V1
 
-while still having ``LIBGS`` set to the correct path, of course. If ``dvisvgm``
+(while still having ``LIBGS`` set to the correct path, of course.) If ``dvisvgm``
```
Comment (behackl) [inline docs/source/installation/troubleshooting.rst:178]: Not sure I agree with the parentheses here, but that is more of a personal remark. No change required.

File: docs/source/installation/troubleshooting.rst [hunk 5]
```diff
@@ -193,31 +195,29 @@ Follow the steps in `this StackExchange answer <https://superuser.com/questions/
 --------------------------------
 
 If ``choco install manimce`` failed,
-it is likely being caused by Python not being added to your ``PATH`` variable properly.
+it is because Python was not added to your ``PATH`` variable properly.
```
Comment (behackl) [inline docs/source/installation/troubleshooting.rst:198]: This does change the meaning somewhat, and I'm not sure the statement is still true after the change. This should be updated to reflect that if ``choco install manimce`` it does not **always** have to be this particular problem.

File: docs/source/installation/troubleshooting.rst [hunk 6]
```diff
@@ -193,31 +195,29 @@ Follow the steps in `this StackExchange answer <https://superuser.com/questions/
...
 Try running the following commands in your terminal:
 
 1. ``py --version``
 2. ``python --version``
 3. ``py3 --version``
 4. ``python3 --version``
 
-Minimally, ``py --version`` and ``python --version`` should return a version.
+Ideally, ``py --version`` and ``python --version`` should return a version.
```
Comment (behackl) [inline docs/source/installation/troubleshooting.rst:206]: I think "minimally" here is used as "out of these four commands, at least the specified two should return a version, but the others could do so as well". Ideally sort of is a stronger, not necessarily intended meaning, no? Do you have any thoughts about that?

**Review by behackl — CHANGES_REQUESTED** [review]
Thank you very much for your contribution! Most changes look fine to me, there are some minor issues regarding wording and formatting that I've raised below; please take a look. Other than that, this is ready to be merged.

File: docs/source/installation/troubleshooting.rst (hunk 5 above)
Comment (naveen521kk) [reply docs/source/installation/troubleshooting.rst:198]: I think this part of troubleshooting can be removed. In the recent version the installation script searches the windows registry for python and doesn't depend on having it in the PATH variable.

File: docs/source/installation/troubleshooting.rst (hunk 6 above)
Comment (peaceheis) [reply docs/source/installation/troubleshooting.rst:206]: Ah, that makes sense. I agree!

File: docs/source/installation/troubleshooting.rst (hunk 5 above)
Comment (peaceheis) [reply docs/source/installation/troubleshooting.rst:198]: I'll go ahead and remove this line if it's superfluous, then.

Comment (peaceheis) [reply docs/source/installation/troubleshooting.rst:198]: Hm... a large section (lines 188t to least 213) refers to the PATH variable... which should I remove and which should I keep? For right now, I'll just update it to say likely.

File: docs/source/installation/troubleshooting.rst [hunk 7]
```diff
@@ -65,15 +67,15 @@ them with:
...
+If you cloned manim from GitHub, you should either remove it
 or run manim outside that folder.
 
 Other errors
 ************
 
 ``pip install manim`` fails when installing manimpango?
 -------------------------------------------------------
-Most likely this means that pip was not able to use our pre-built wheels
+This mostly means that pip was not able to use our pre-built wheels
```
Comment (Darylgolden) [inline docs/source/installation/troubleshooting.rst:78]:
```suggestion
This most likely means that pip was not able to use our pre-built wheels
```

**Review by Darylgolden — CHANGES_REQUESTED** [review]
Minor change, otherwise LGTM!

**Review by Darylgolden — APPROVED** [review]

**Pull request merged**